with-electron/README.md

```diff
@@ -1,14 +1,22 @@
 # Electron Example
 
+<p>
+  <!-- Web -->
+  <img alt="Supports Expo Web" longdesc="Supports Expo Web" src="https://img.shields.io/badge/web-4630EB.svg?style=flat-square&logo=GOOGLE-CHROME&labelColor=4285F4&logoColor=fff" />
```

Comment:
I only added web, since electron is powered by the web part 😄 But if android/ios needs to be added, lmk